tests/xfstests/install.pm

```diff
@@ -45,6 +45,16 @@ sub run {
     zypper_call('--gpg-auto-import-keys ref', timeout => 600);
     zypper_call('in qa_test_xfstests',        timeout => 1200);
 
+    # Install upstream fio if not find usable fio in current repos
+    if (script_run('zypper se -x fio') == 0) {
```

Comment:
As we have fio fully available in SLES, this check and following lines are not needed.
Also SLE12SP5 verification runs are not complete.